R/helper-junitmock.R

```diff
@@ -0,0 +1,18 @@
+createJunitReporterMock <- function (file) {
```

Comment:
Could you add a brief comment along the lines of "Fix components of JunitReporter that otherwise vary from run-to-run"